packages/runtime-common/worker.ts

```diff
@@ -165,14 +165,16 @@ export class Worker {
   async run() {
     await this.#queue.start();
 
-    this.#queue.register(
-      `from-scratch-index:${this.#realmURL}`,
-      this.fromScratch,
-    );
-    this.#queue.register(
-      `incremental-index:${this.#realmURL}`,
-      this.incremental,
-    );
```

Comment:
whoops! missing `await`'s here...

Reply:
should we try [`require-await`](https://typescript-eslint.io/rules/require-await/)?

Reply:
that would not have caught this since there is an await elsewhere in this function, it's just that it was forgotten for these 2 statements